src/formatter/createPropFilter.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+
+export default function createPropFilter(
+  props: {},
+  filter: string[] | ((any, string) => boolean)
+) {
+  if (Array.isArray(filter)) {
+    return (key: string) => filter.indexOf(key) === -1;
+  } else {
+    return (key: string) => filter(props[key], key);
```

Comment:
Do we agree on the filter function being called with the prop value as the first argument, and the prop key as the second? I did it that way only because that's how most of lodash's functions work ([lodash#filter](https://lodash.com/docs/4.17.15#filter), [lodash#map](https://lodash.com/docs/4.17.15#map)), and I'm used to lodash.

If the string array form is really going to be deprecated, then it might make more sense for the key to be the first argument here. Then you wouldn't have to write your filter function with an unused first arg if you just wanted to filter by key.